classes/WP_CLI/ActionScheduler_WPCLI_System_Command.php

```diff
@@ -0,0 +1,220 @@
+<?php
+
+// phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped -- Escaping output is not necessary in WP CLI.
+
+use function \WP_CLI\Utils\get_flag_value;
+
+/**
+ * System info WP-CLI commands for Action Scheduler.
+ */
+class ActionScheduler_WPCLI_System_Command {
```

Comment:
Is it intentional to use this class naming convention here, whereas sibling class `Action_Command` is now namespaced?

Reply:
It was, but I'll change it.